hugegraph-rocksdb/src/main/java/com/baidu/hugegraph/backend/store/rocksdb/RocksDBStore.java

```diff
@@ -444,36 +444,26 @@ private void mutate(Session session, BackendAction item) {
         BackendEntry entry = item.entry();
 
         RocksDBTable table;
+        if (entry.type() != HugeType.OLAP && !entry.olap()) {
```

Comment:
not the same between `entry.olap()` and `entry.type() == HugeType.OLAP`?

Reply:
not same. `entry.type() == HugeType.OLAP` means olap vertex, but `entry.olap()` means olap vertex index